Packs/CIRCL/ReleaseNotes/1_0_27.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### CIRCL CVE Search
+
+- The cve_latest command is deprecated – we added support for 5.1 format and redirected users to the NVD Feed 2.0 integration.
```

Comment:
- Deprecated ***cve_latest*** command - Use NVD Feed 2.0 instead.
- Updated the ***cve*** command to support the new 5.1 format.

Please go over https://xsoar.pan.dev/docs/documentation/release-notes